skins/oasis/css/core/PageHeader.scss

```diff
@@ -44,13 +48,11 @@
 		display: inline;
 		font-size: 26px;
```

Comment:
:question: does https://github.com/WikiaVolDev/app/commit/4ff5617491d6aa833edc1e3601fbb97897aebcc3#diff-5a1adbc1aca329ef8b292df01c02a86eR314 have anything to do with this one?